Meadowlark-js/packages/meadowlark-core/src/middleware/AuthenticationMiddleware.ts

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { writeDebugStatusToLog, writeRequestToLog } from '../Logger';
+import { validateJwt } from '../security/JwtValidator';
+import { authorizationHeader } from '../security/AuthorizationHeader';
+import { MiddlewareChain } from './MiddlewareChain';
+
+const moduleName = 'AuthenticationMiddleware';
+
+/**
+ * Handles authentication
+ */
+export async function authenticate({ frontendRequest, frontendResponse }: MiddlewareChain): Promise<MiddlewareChain> {
+  // if there is a response already posted, we are done
+  if (frontendResponse != null) return { frontendRequest, frontendResponse };
```

Comment:
Example of a middleware passing through if there was a response from a previous middleware